gr-blocks/python/blocks/qa_message.py

```diff
@@ -58,8 +58,8 @@ def test_100(self):
         self.assertEquals(0, msg.length())
 
     def test_101(self):
-        s = 'This is a test'
-        msg = gr.message_from_string(s)
+        s = b'This is a test'
+        msg = gr.message_from_string(s.decode('utf8'))
         self.assertEquals(s, msg.to_string())
```

Comment:
Could you also test the other way around, throw in `s=unicode(…)`, and compare the result after encoding to utf8? This feels like something that will fall on our feet if we don't check stuff. And also, there's so many nice characters in UTF, why stick with ASCII? Why not 🦀?

Reply:
:+1: I'll throw in more tests for "nice" unicode characters :)

Reply:
s = unicode() is Py2 only code. `u"mystring"` is how you do unicode (with unicode_literals imported from __future__)

Reply:
Now testing is in place and should check all boxes (Had to explictely tell python about the utf-8 encoding of the file and import unicode_literals to have the same code for py2/py3). If people wish to use unicode in py2 they definitely have to do the same thing.